lib/api_entity.rb

```diff
@@ -98,19 +98,11 @@ def collection(collection_path, opts = {})
       retries = 0
       begin
         resp = api.get(collection_path, opts)
-        case resp.status
-        when 404
-          raise ApiEntity::NotFound, TariffJsonapiParser.new(resp.body).errors
```

Comment:
Can we remove the defined ApiEntity::NotFound error, now?

Reply:
And the other ones we've removed?

Reply:
Yeh - ApiEntity::Error is still referenced elsewhere but I'll remove the others